dev-python/pipenv/pipenv-2020.6.2.ebuild

```diff
@@ -11,19 +12,22 @@ MY_PV=${PV/_beta/b}
 DESCRIPTION="Python Development Workflow for Humans"
 HOMEPAGE="https://github.com/pypa/pipenv https://pypi.org/project/pipenv/"
 SRC_URI="https://github.com/pypa/pipenv/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"/${PN}-${MY_PV}
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE="test"
+RESTRICT="!test? ( test )"
 
 RDEPEND="
 	dev-python/certifi[${PYTHON_USEDEP}]
 	>=dev-python/pip-18.0[${PYTHON_USEDEP}]
 	dev-python/virtualenv[${PYTHON_USEDEP}]
 	>=dev-python/virtualenv-clone-0.2.5[${PYTHON_USEDEP}]
-	>=dev-python/setuptools-36.2.1[${PYTHON_USEDEP}]"
-DEPEND="test? (
+	"
+
+BDEPEND="test? (
```

Comment:
Move `test?` into next line to have indent readable.

Reply:
Done. I hope this is fine now.